doc/source/train/user-guides/configuration-overview.rst

```diff
@@ -1,32 +0,0 @@
-:orphan:
-
-Configuration Overview
-======================
-
-.. _train-run-config:
```

Comment:
It looks like this is being (incorrectly) referenced in this link [here](https://github.com/ray-project/ray/blame/86c0958a5f051780e1f4cf08ad37bde942040774/doc/source/train/huggingface-accelerate.rst#L209). Can you update it to point to the persistent storage guide?

Reply:
Done and verified in `file:///Users/tseah/ray/doc/_build/html/index.html`